SocialCareCaseViewerApi/Startup.cs

```diff
@@ -43,7 +34,11 @@ public static void ConfigureServices(IServiceCollection services)
             services
                 .AddMvc()
                 .SetCompatibilityVersion(CompatibilityVersion.Version_3_0)
-                .AddFluentValidation();
+                .AddFluentValidation()
+                .AddJsonOptions(options =>
+                {
+                    options.JsonSerializerOptions.IgnoreNullValues = true;
+                });
```

Comment:
This is the change to make null properties not be returned in our JSON response

Reply:
Would the front end need to be aware of which properties are not going to be returned. Would it be expecting a specific object and somehow know what to do if a specific property does not exist?